CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+x.xx.x Release notes (yyyy-MM-dd)
+=============================================================
+
+### API breaking changes
+
+* None.
+
+### Enhancements
+
+* Reduce the amount of memory used by RLMRealm notification listener threads.
```

Comment:
Small change, but maybe replace "RLMRealm" with "Realm" to make more language agnostic?